app/index.js

```diff
@@ -66,8 +62,8 @@ var NemoGenerator = yeoman.generators.Base.extend({
       fs.writeFileSync('Gruntfile.js', gruntContent);
     }
     //let's update Gruntfile.js if possible
-    var gruntfileData = fs.readFileSync('Gruntfile.js'),
-      output = gruntFileApi.init(gruntfileData);
+    var gruntfileData = fs.readFileSync('Gruntfile.js');
```

Comment:
I would say, if you are going to remove this grouped `var` declaration, please can you do the same throughout this file?